packages/site/src/components/PaymentChannelContainer.tsx

```diff
@@ -38,7 +50,17 @@ export default function PaymentChannelContainer({
         focusedPaymentChannel={focusedPaymentChannel}
         setFocusedPaymentChannel={setFocusedPaymentChannel}
       />
-      <PaymentChannelDetails paymentChannel={focusedPaymentChannel} />
+      <div className={classes.paymentDetails}>
+        <PaymentChannelDetails
+          channelID={focusedPaymentChannel}
+          counterparty={"0x123"}
+          capacity={1000}
+          myBalance={150}
+          status={"Proposed" as ChannelStatus}
```

Comment:
This hard-coded data can be replaced with data from the supplied `PaymentChannelInfo`. I'm not too familiar with react hooks, but can `focusedPaymentChannel` be typed as the richer `PaymentChannelInfo` instead of just the ID string? Alternate: just update the index of the focused channel, and then do something like 

```
<PaymentChannelDetails
  channelID={paymentChannels[focusedPaymentChannel].ID}
  counterparty={paymentChannels[focusedPaymentChannel].Balance...} // check vs myAddress to see which person is the counterparty
  capacity={paymentChannels[focusedPaymentChannel].Balance...} // add up PaidSoFar + RemainingFunds
  myBalance={paymentChannels[focusedPaymentChannel].Balance...}
  status={paymentChannels[focusedPaymentChannel].Status}
/>
```

Would be great to get `PaymentChannelContainer.stories.tsx` up with a few channels passed in - so that clicking between the channels in the list updates the rendered `PaymentChannelDetails`.